### PR TITLE
Task00 Leonid Petrov SPbU

### PR DESCRIPTION
**Pull request by lpetrov02**

### CODE REVIEW

**Comment by lpetrov02** [comment]
Task00 Леонид Петров SPbU

<details><summary>Локальный вывод</summary><p>

<pre>
$ ./enumDevices
Number of OpenCL platforms: 1
Platform #1/1
    Platform name: Intel(R) CPU Runtime for OpenCL(TM) Applications
    Platform vendor: Intel(R) Corporation
    Platform's devices found: 1
    Device #1/1
        Device name: AMD A12-9700P RADEON R7, 10 COMPUTE CORES 4C+6G
        Device type: CPU
        Device local memory size: 32768
        Device max work group size: 8192
        Device max work group sizes: (8192, 8192, 8192, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, )
</pre>

</p></details>

<details><summary>Вывод Github CI</summary><p>

<pre>
$ ./enumDevices
Number of OpenCL platforms: 1
Platform #1/1
    Platform name: Intel(R) CPU Runtime for OpenCL(TM) Applications
    Platform vendor: Intel(R) Corporation
    Platform's devices found: 1
    Device #1/1
        Device name: Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
        Device type: CPU
        Device local memory size: 32768
        Device max work group size: 8192
        Device max work group sizes: (8192, 8192, 8192, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, )
</pre>

</p></details>

File: src/main.cpp [hunk 1]
```diff
@@ -64,27 +68,87 @@ int main() {
         // Затем откройте документацию по clGetPlatformInfo и в секции Errors найдите ошибку, с которой столкнулись
         // в документации подробно объясняется, какой ситуации соответствует данная ошибка, и это позволит, проверив код, понять, чем же вызвана данная ошибка (некорректным аргументом param_name)
         // Обратите внимание, что в этом же libs/clew/CL/cl.h файле указаны всевоможные defines, такие как CL_DEVICE_TYPE_GPU и т.п.
+        // ANSWER 1.1: Попробовал заменить CL_PLATFORM_NAME на число 30, получил ошибку -30 (INVALID_VALUE, это и означает некорректный идентификатор платформы)
```
Comment (simiyutin) [inline src/main.cpp:71]: :+1:

File: src/main.cpp [hunk 2]
```diff
@@ -5,6 +5,10 @@
 #include <sstream>
 #include <stdexcept>
 #include <vector>
+#include <map>
+
+
+#define Mb 1024 * 1024
```
Comment (simiyutin) [inline src/main.cpp:11]: Минорный комментарий что по дефолту лучше пользоваться константами вместо дефайнов

**Comment by simiyutin** [comment]
Отлично, задача зачтена, 5/5 баллов :+1:

**Pull request closed without merging**